addons/binding/org.openhab.binding.networkcamera/src/main/java/org/openhab/binding/networkcamera/handler/NetworkCameraHandler.java

```diff
@@ -10,7 +10,12 @@
 
 import static org.openhab.binding.networkcamera.NetworkCameraBindingConstants.*;
 
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
Please do not use `@NonNull`, see https://www.eclipse.org/smarthome/documentation/development/conventions.html